package.json

```diff
@@ -10,28 +10,34 @@
   "repository": "git@github.com:TwigWorld/eslint-config-twig.git",
   "author": "Adam Oliver <mail@adamoliver.net>",
   "license": "MIT",
-  "dependencies": {
-    "babel-eslint": "7",
-    "eslint-config-airbnb": "18.2.1",
-    "eslint-plugin-compat": "^3.8.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-no-only-tests": "^2.3.1",
-    "eslint-plugin-react": "^7.21.5",
-    "eslint-plugin-react-hooks": "4.2.0"
-  },
   "devDependencies": {
-    "eslint": "7.2.0"
+    "@babel/core": "^7.22.20",
+    "@babel/eslint-parser": "^7.22.15",
+    "eslint": "^8.49.0",
+    "eslint-config-airbnb": "18.2.1",
```

Comment:
This should probably have a carat

Reply:
👍  I've pushed up a new commit to include this